std_msgs/Header header
float64 left_stick_y
float64 right_stick_y
float64 amplitude
float64 frequency
int32 mode_switch

